src/battle.py

```diff
@@ -209,7 +293,7 @@ def handle_ranged(self, d, proj, best_material, stats, enemy_stats):
 
                 if self.canvas:
                     color = self.canvas.itemcget(self.id, 'fill')
-                    proj[-1].id = self.canvas.create_oval(self.x, self.y, self.x + PROJECTILE_RADIUS, self.y + PROJECTILE_RADIUS, width=0, fill=color)
+                    proj[-1].id = self.canvas.create_oval(self.x, self.y, self.x + 2, self.y + self.get_projectile_size(), width=0, fill=color)
```

Comment:
Why is there `x + 2` but `y + self.get_projectile_size()`?

Reply:
I tried to implement something.... but it's not working at the moment.